eureka/S5_lightcurve_fitting/fitters.py

```diff
@@ -1213,23 +1238,25 @@ def save_fit(meta, lc, model, fitter, fit_params, freenames, samples=[],
         The list of fitted parameter names.
     samples : ndarray; optional
         The full chain from a sampling method, by default [].
-    upper_errs : ndarray; optional
-        The one sigma upper limits from a sampling method, by default [].
-    lower_errs : ndarray; optional
-        The one sigma lower limits from a sampling method, by default [].
+    spec_table: astropy table; optional
+        Transmission spectrum (now only produced in S6)
+    Notes
+    -----
+    History:
+
+    - Mar 13-Apr 18, 2022 Caroline Piaulet
+         Record an astropy table for mean, median, percentiles,
+             +/- 1 sigma, all params
+         Save transmission spectrum (commented out)
```

Comment:
This history line (and the similar one for the emceefitter function) are indented by one too many spaces (should only be 4 spaces ahead of the start of the dash), and you can't have the "+/- ..." line be extra indented; both of these cause warnings while building the docs. "Save transmission spectrum (commented out)" can also be removed from both history lines.